### PR TITLE
Changed the sentence to make it easier to read

### PR DESCRIPTION
**Pull request by alexr1997**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/sql/dql/basic-selects/select-all.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/sql/dql/basic-selects/select-all.md?ref=c93362ed366b83f46820b4231b4915ac8a2e8f84)

**Comment by kirillongithub** [comment]
suggest editing two paragraphs from this to the following (note that you can't increase readability...just improve it):

--

The query above `SELECT`s all the columns (represented by the `*` symbol), in the order of their definition in the database, `FROM` the table named `pokemon`. The returned table will contain all of the *columns* and all of the *rows* from the table.

Note that all the SQL keywords like `SELECT` and `FROM` are case insensitive, but they are capitalized to improve readability. The example above can also be written as:

File: sql/dql/basic-selects/select-all.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ SELECT *
 FROM pokemon;
 ```
 
-What the above query does is that it `SELECT`s all (represented by the `*` symbol) the columns in the order of their definition in the database `FROM` the table named `pokemon`. The returned table will contain all of the *columns* and all of the *rows* from the table.
+The above query `SELECT`s all (represented by the `*` symbol) the columns in the order of their definition in the database `FROM` the table named `pokemon`. The returned table will contain all of the *columns* and all of the *rows* from the table.
```
Comment (kapnobatai137) [inline sql/dql/basic-selects/select-all.md]:
```suggestion
The query above `SELECT`s all the columns (represented by the `*` symbol), in the order of their definition in the database, `FROM` the table named `pokemon`. The returned table will contain all of the *columns* and all of the *rows* from the table.
```

File: sql/dql/basic-selects/select-all.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ SELECT *
...
 
 Note that all the SQL keywords like `SELECT` and `FROM` are case insensitive, but they are capitalized to increase readability. The example above can also be written as:
```
Comment (kapnobatai137) [inline sql/dql/basic-selects/select-all.md]:
```suggestion 
Note that all the SQL keywords like `SELECT` and `FROM` are case insensitive, but they are capitalized to improve readability. The example above can also be written as:
```

**Pull request merged**